node_modules/ioslib/lib/simulator.js

```diff
@@ -20,6 +22,7 @@ const mkdirp = require('mkdirp');
 const net = require('net');
 const path = require('path');
 const readPlist = require('./utilities').readPlist;
+const simctl = require('./simctl');
```

Comment:
It was not committed because last October a line was added to the `.gitignore` to ignore everything [new] in the `node_modules` directory. I fixed it.